### DISCUSSION TITLE
求助

### DISCUSSION THREAD

**Discussion by TOKLK** [share121/inter-knot#2086] [其他]
![Screenshot_2024-08-14-05-36-39-750_mark via](https://github.com/user-attachments/assets/cd10006d-9d28-4569-a558-545d78ce1a9f)
为什么有时候发帖或者回帖的时候会这样啊

**Comment by LANKgilu** [comment]
我不到啊

**Reply by TOKLK** [reply]
你也会出现这种情况吗？

**Reply by LANKgilu** [reply]
> 你也会出现这种情况吗？有时候会
